packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -57,6 +57,7 @@ class Braze {
       baseUrl: this.endPoint,
       allowUserSuppliedJavascript: this.allowUserSuppliedJavascript,
     });
+    window.braze.addSdkMetadata([window.braze.BrazeSdkMetadata.CDN, 'rud']);
```

Comment:
I'd rather suggest to use our full company name, `rudderstack`.

@krishna2020, I remember we send similar information in other integrations (cloud/device) as well. Is there a convention we follow?
CC @koladilip

Reply:
This is an internal key, not visible to customers and we can't change it.